modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1110,24 +1165,34 @@ def resample_agg_df(self, resample_kwargs, func, *args, **kwargs):
 
     def resample_transform(self, resample_kwargs, arg, *args, **kwargs):
         return self._resample_func(
-            resample_kwargs, "transform", arg=arg, *args, **kwargs
+            resample_kwargs, "transform", arg=arg, use_range_impl=False, *args, **kwargs
```

Comment:
[this approach](https://github.com/modin-project/modin/pull/7140/files#r1549246505) doesn't work well with transform operations, so all of them are disabled